lms/djangoapps/certificates/queue.py

```diff
@@ -351,8 +351,7 @@ def add_cert(self, student, course_id, forced_grade=None, template_file=None, ge
         # analytics. Only do this if the certificate is new, or
         # already marked as ineligible -- we don't want to mark
         # existing audit certs as ineligible.
-        cutoff = settings.AUDIT_CERT_CUTOFF_DATE
-        if (cutoff and cert.created_date >= cutoff) and not is_eligible_for_certificate:
+        if not is_eligible_for_certificate:
```

Comment:
So... I won't lie, I didn't put a ton of thought into this. My aim was to remove references to `AUDIT_CERT_CUTOFF_DATE`. 

This code is no longer being executed and is part of v1 certificate generation. We have another ticket in progress (MICROBA-1227) that includes deleting `queue.py` as part of the removing v1 cert code. I ended up doing this work before `queue.py` was removed.